app/assets/stylesheets/public_view_shop.sass

```diff
@@ -0,0 +1,65 @@
+$semi-white: #FF9495
+
+#public-view-shop-header
+  // background-color: $white
+  .card-block
+    background-color: $white
+
+    .text-xs-center
+
+      .card-title
+        margin-top: -60px
+        margin-bottom: 40px
+        padding-left: 70px
+
+      .btn
+        background-image: linear-gradient(to right, $blue 0%, #187CB5 100%)
+        background-color: $blue
+        border: 1px solid $blue
+        display: inline-block
+        padding: 10px
+        line-height: 1.5
+        border-radius: 0.25rem
+        width: 50%
+
+#public-view-shop-body
+  background-color: $white
+
+  .m-y-2
+    width: 55%
```

Comment:
Mhm didn't think it would actually overwrite bootstrap. I will add in another class name now.